### PR TITLE
Sharpened the scalpels! (Scalpels can now be used to butcher.)

### PR DESCRIPTION
**Pull request by QuietlyWhisper**
## Sharpened the scalpels! (Scalpels can now be used to butcher.) 

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Whisper
- add: Scalpels can now be used to butcher in emergencies, 50% slower than standard tools.

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Specific/Medical/surgery.yml
Comment (DrSmugleaf) [inline Resources/Prototypes/Entities/Objects/Specific/Medical/surgery.yml:58]: This won't get read properly unless the indentation is correct, `butcherDelayModifier` is a property of `SharpComponent` so it needs to be at the same indentation level.
```suggestion
  - type: Sharp
    butcherDelayModifier: 1.5 # Butchering with a scalpel, regardless of the type, will take 50% longer
```

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Entities/Objects/Specific/Medical/surgery.yml
Comment (QuietlyWhisper) [reply Resources/Prototypes/Entities/Objects/Specific/Medical/surgery.yml]: Thanks for the assistance, I wasn't aware indents were part of the syntax!

**Comment by QuietlyWhisper** [comment]
Should be good to go now!

**Review by DrSmugleaf — APPROVED** [review]

**Comment by QuietlyWhisper** [comment]
What's up with the workflows awaiting approval? Is that on my end or a maintainer?
DrSmugLeaf approved the changes and that seems to be all I'm waiting on to merge-

**Comment by mirrorcult** [comment]
> What's up with the workflows awaiting approval? Is that on my end or a maintainer? DrSmugLeaf approved the changes, and that seems to be all I'm waiting on to merge-

first-time contributors don't have actions run without approval

**Comment by QuietlyWhisper** [comment]
Here we are, approved, all checks passed, no conflicts and ready to merge!

**Review by vulppine — APPROVED** [review]

**Pull request merged**
